components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/util/APIClient.java

```diff
@@ -163,7 +165,8 @@ private void handleSuccess(ActionInvocationResponse.Builder builder, HttpEntity
         try {
             builder.response(handleSuccessResponse(entity));
         } catch (ActionInvocationException e) {
-            builder.errorLog("Unexpected response for status code: " + statusCode + ". " + e.getMessage());
+            builder.errorLog("Unexpected response for status code " + statusCode
+                    + ". " + e.getMessage());
```

Comment:
can take this to one line I guess

Reply:
Also be consistent on using the colon or not using it

Reply:
Updated in